common/dna_data/wc_human_dna.txt

```diff
@@ -2699,4 +2699,608 @@ aiden_perenolde_dna = { # by ercarp
+ 		gene_bs_nose_ridge_width={ "nose_ridge_width_neg" 31 "nose_ridge_width_pos" 22 }
+ 		gene_bs_nose_size={ "nose_size_neg" 115 "nose_size_pos" 10 }
+ 		gene_bs_nose_tip_angle={ "nose_tip_angle_neg" 13 "nose_tip_angle_neg" 27 }
+ 		gene_bs_nose_tip_forward={ "nose_tip_forward_neg" 83 "nose_tip_forward_neg" 17 }
+ 		gene_bs_nose_tip_width={ "nose_tip_width_neg" 63 "nose_tip_width_pos" 65 }
+ 		face_detail_cheek_def={ "cheek_def_01" 197 "cheek_def_02" 98 }
+ 		face_detail_cheek_fat={ "cheek_fat_01_pos" 4 "cheek_fat_02_pos" 94 }
+ 		face_detail_chin_cleft={ "chin_cleft" 8 "chin_dimple" 10 }
+ 		face_detail_chin_def={ "chin_def" 81 "chin_def" 7 }
+ 		face_detail_eye_lower_lid_def={ "eye_lower_lid_def" 60 "eye_lower_lid_def" 71 }
+ 		face_detail_eye_socket={ "eye_socket_01" 198 "eye_socket_02" 211 }
+ 		face_detail_nasolabial={ "nasolabial_01" 104 "nasolabial_02" 123 }
+ 		face_detail_nose_ridge_def={ "nose_ridge_def_pos" 157 "nose_ridge_def_neg" 95 }
+ 		face_detail_nose_tip_def={ "nose_tip_def" 193 "nose_tip_def" 10 }
+ 		face_detail_temple_def={ "temple_def" 212 "temple_def" 216 }
+ 		expression_brow_wrinkles={ "brow_wrinkles_04" 33 "brow_wrinkles_04" 33 }
+ 		expression_eye_wrinkles={ "eye_wrinkles_01" 62 "eye_wrinkles_01" 62 }
+ 		expression_forehead_wrinkles={ "forehead_wrinkles_01" 135 "forehead_wrinkles_01" 135 }
+ 		expression_other={ "cheek_wrinkles_both_01" 0 "cheek_wrinkles_both_01" 0 }
+ 		complexion={ "complexion_2" 70 "complexion_6" 89 }
+ 		gene_height={ "normal_height" 121 "normal_height" 122 }
+ 		gene_bs_body_type={ "body_fat_head_fat_low" 111 "body_fat_head_fat_medium" 148 }
+ 		gene_bs_body_shape={ "body_shape_average_clothed" 102 "body_shape_average" 0 }
+ 		gene_bs_bust={ "bust_clothes" 161 "bust_shape_2_full" 161 }
+ 		gene_age={ "old_4" 48 "old_4" 48 }
+ 		gene_eyebrows_shape={ "avg_spacing_high_thickness" 136 "avg_spacing_lower_thickness" 135 }
+ 		gene_eyebrows_fullness={ "layer_2_high_thickness" 158 "layer_2_low_thickness" 191 }
+ 		gene_body_hair={ "body_hair_avg" 90 "body_hair_avg" 90 }
+ 		gene_hair_type={ "hair_wavy" 174 "hair_wavy" 174 }
+ 		gene_baldness={ "no_baldness" 127 "no_baldness" 127 }
+ 		eye_accessory={ "normal_eyes" 108 "normal_eyes" 108 }
+ 		teeth_accessory={ "normal_teeth" 0 "normal_teeth" 0 }
+ 		eyelashes_accessory={ "normal_eyelashes" 46 "normal_eyelashes" 46 }
+ 		gene_bs_ear_lenght={ "floppy_ear_length" 0 "floppy_ear_length" 0 }
+ 		special_eyebrows={ "no_eyebrows" 182 "no_eyebrows" 182 }
+ 		special_eyes={ "no_special_eyes" 0 "no_special_eyes" 0 }
+ 		gene_bs_mouth_tusks={ "mouth_fangs" 0 "mouth_fangs" 0 }
+ 		static_tusks={ "no_static_tusks" 0 "no_static_tusks" 0 }
+ 		gene_bs_ear_tip={ "ear_tip_pos" 0 "ear_tip_pos" 0 }
+ 		gene_bs_fingers={ "fingers_troll" 0 "fingers_troll" 0 }
+ 		gene_skin_value={ "skin_value_pos" 0 "skin_value_neg" 0 }
+ 		gene_race={ "creature_human" 240 "creature_human" 240 }
+		}
+	}
+	enabled=yes
+}
+
+taelan_fordring_dna = {
```

Comment:
He has red hairs? What did you use for a reference?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/123141327-535a1800-d469-11eb-9097-aa6a1fea201c.png)

</details>

Reply:
he has 3 different colors in his picture, brown, blonde and red hair which should i use

Reply:
DNA will be reworked on the `dna-2` branch.

62dd73f68a2e1f45710f41f5537d3395321dd192